jobs/config.json

```diff
@@ -1783,6 +1783,18 @@
     "--env-file=platforms/gke.env",
     "--env-file=jobs/ci-kubernetes-e2e-gke-prod-smoke.env"
   ]
+},
+
+"ci-kubernetes-e2e-gke-1.3-1.4-upgrade-master": {
+  "scenario": "kubernetes_e2e",
+  "args": [
+    "--env-file=platforms/gke.env",
+    "--env-template=jobs/template-e2e-upgrade-master.env",
+    "--env=PROJECT=kube-gke-upg-1-3-1-4-upg-mas",
```

Comment:
Would it make sense to create a gke-1.3-1.4-upgrade-master.env that defines these 4 args and put that before the template.env?